src/ConformalTracking.cc

```diff
@@ -88,6 +85,33 @@ ConformalTracking::ConformalTracking() : Processor("ConformalTracking") {
                            "Name of the TrackerHit input collections from the Vertex Endcap", m_inputVertexEndcapCollections,
                            {"VXDEndcapTrackerHits"});
 
+  // Parameters for tracking
+  registerProcessorParameter("MaxCellAngle", "Cut on angle between two cells for cell to be valid", m_maxCellAngle,
+                             double(0.035));
+  registerProcessorParameter("MaxCellAngleRZ", "Cut on angle between two cells in RZ for cell to be valid", m_maxCellAngleRZ,
+                             double(0.035));
+  registerProcessorParameter("MaxDistance", "Maximum length of a cell (max. distance between two hits)", m_maxDistance,
+                             double(0.015));
+  registerProcessorParameter("HighPTCut", "pT threshold (in GeV) for enabling extendHighPT in extendTracks", m_highPTcut,
+                             double(10.0));
+  registerProcessorParameter("MaxChi2", "Maximum chi2/ndof for linear conformal tracks", m_chi2cut, double(300.));
+  registerProcessorParameter("MinClustersOnTrack", "Minimum number of clusters to create a track in pattern recognition",
+                             m_minClustersOnTrack, int(6));
+
+  registerProcessorParameter("MaxChi2Increase", "Chi2 increase when adding new hits to a track", m_chi2increase,
```

Comment:
It seems to me that this parameter is not set for each step. Is it true and wanted? If so, the variable `m_chi2increase` is set to 0 when `ConformalTrackingV2` is run.
@andresailer @eleogran

Reply:
It seems this parameter is not used at all

Reply:
Should I then remove it?

Reply:
Yes